pkg/apis/crd/v1alpha1/types.go

```diff
@@ -600,3 +605,18 @@ type NamespacedName struct {
 	Name      string `json:"name,omitempty"`
 	Namespace string `json:"namespace,omitempty"`
 }
+
+// NetworkPolicyProtocol includes all protocols that `ports` can't support. All
+// fields should be used as a stand-alone field. To match all traffic with a
```

Comment:
standalone field

What does "standalone field" mean?

Reply:
It means all fields cannot be used with each other.